packages/core/src/components/menu/menu.tsx

```diff
@@ -599,7 +599,7 @@ export class Menu {
         }}
         slot="menu"
       >
-        <aside
+        <nav
```

Comment:
Is this change necessary? Should the aside stay and the <div id="menu-tabs" be replaced with nav? (https://www.w3.org/TR/2011/WD-html5-author-20110809/the-aside-element.html, https://stackoverflow.com/questions/40390390/semantic-differences-between-nav-and-aside-for-navigation-in-html)